.gitignore

```diff
@@ -61,10 +62,12 @@ run-local.bat
 docker_local.sh
 appsettings.seawolf.json
 appsettings.shelby.json
+appsettings.RIK-LAPTOP.json
```

Comment:
Machine specifieke appsettings nooit inchecken

Reply:
Is het juist niet de bedoeling dat die in de gitignore komen? Anders is er meer risico dat er ene per ongeluk mee gaat. Zoals bvb die `seawolf` en `shelby`. Anders moeten we een `local` variant voorzien dat wordt ge-gitignored.

Reply:
gitignore is aangepast om alle machine specific jsons te ignoren